doc/source/operations.rst

```diff
@@ -55,23 +55,60 @@ Once the appropriate profile has been selected, it can be applied by running the
 Updates
 =======
 
-Update Airship UCP Services
----------------------------
+SUSE Containerized OpenStack is delivered as an rpm package, so performing updates is generally accomplished by simply updating the rpm package to the latest version and redeploying the cloud using the steps outlined in the installation procedure. This is the typical update path and will incorporate all recent changes, as well as automatically updating component chart and image versions. However, there may be use cases where it is desirable to update services or components to specific versions. 
+
+Updating OpenStack Version
+--------------------------
+
+To make a global change to the OpenStack version used by all component images, create a key in ${WORKSPACE}/env/extravars called "suse_osh_image_version" and set it to the desired value. For example, to use the "ocata" version, add the following line to the extravars file:
+
+.. code-block:: yaml
+
+   suse_osh_image_version: "ocata"
+
+It is also possible to update an individual image or subset of images to a different version, rather than making a global change. To do this, it will be necessary to manually edit the versions.yaml file located in socok8s/site/soc/software/config/. Locate the images that need to be changed in the "images" section of the file and modify the line to include the desired version. For example, to use the "rocky" version for the heat_api image, change the following line in versions.yaml from
+
+.. code-block:: yaml
+
+   heat_api: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+
+to
+
+.. code-block:: yaml
 
-Update OpenStack Services
--------------------------
+   heat_api: "{{ suse_osh_registry_location }}/openstackhelm/heat:rocky"
 
-Update secrets, passwords and certificates
+Updating Individual Images and Helm Charts
 ------------------------------------------
 
-Update this repository
-----------------------
+The versions.yaml file can also be used for more advanced update configurations such as using a specific image or Helm chart source version. 
 
-The SUSE Containerized OpenStack repository can be updated by performing the git pull command from the socok8s directory:
+.. note::
+   
+   The following configuration options are not supported for the SUSE Containerized OpenStack Technical Preview. Please proceed at your own risk.
```

Comment:
we need words like "A change to a non default image registry and/or custom built image will lose any product supported by SUSE. "